gl/gloffscreen_glx.c

```diff
@@ -95,17 +95,51 @@ GloContext *glo_context_create(int formatFlags)
 #endif
 
     /* Create GLX context */
-    context->glx_context = glXCreateNewContext(x_display, configs[0], GLX_RGBA_TYPE, NULL, True);
+
+    /* FIXME: Check for "GLX_ARB_create_context" extension*/
+    #define GLX_CONTEXT_MAJOR_VERSION_ARB     0x2091
+    #define GLX_CONTEXT_MINOR_VERSION_ARB     0x2092
+    #define GLX_CONTEXT_PROFILE_MASK_ARB     0x9126
+    #define GLX_CONTEXT_CORE_PROFILE_BIT_ARB 0x00000001
```

Comment:
No idea, my GLX file provided all of them, however, chances are not every GLX header has them?
MAJOR and MINOR_VERSION_ARB were defined in all GLX samples I found, the later two not, but when things were crashy I decided to also def them.
Maybe we should check the GLX manual or something.
It was more important to me that xqemu was working again at all.